install/charts/dir/values.yaml

```diff
@@ -146,35 +146,73 @@ apiserver:
       #   bundleEndpointProfile:
       #     type: https_web
 
-  # Extra volume mounts for the apiserver container
-  extraVolumeMounts: 
-  - name: zot-config-storage
-    mountPath: /etc/zot
-
-  # Extra volumes - shared between apiserver and zot for config hot reload
-  extraVolumes: 
-  - name: zot-config-storage
-    hostPath:
-      path: /opt/zot-config
-      type: DirectoryOrCreate
-
   # Zot registry configuration (subchart)
   zot:
-    # Disable default config mounting to use shared volume
+   # Disable default config mounting
     mountConfig: false
-    mountSecret: false
+
+    # Enable default secret mounting
+    mountSecret: true
+
+    # ZOT configuration file
+    configFiles:
+      config.json: |-
+        {
+          "distSpecVersion": "1.1.1",
+          "storage": {
+            "rootDirectory": "/var/lib/registry"
+          },
+          "http": {
+            "address": "0.0.0.0",
+            "port": "5000",
+            "auth": {
+              "htpasswd": {
+                "path": "/secret/htpasswd"
+              }
+            },
+            "accessControl": {
+              "adminPolicy": {
+                "users": ["apiserver"],
```

Comment:
Just a question, because I'm not sure. What is the case if the `HTPASSWD_USERNAME` is not the default `apiserver` (Taskfile)?